### PR TITLE
server, docs: change rowBin to request.

### LINKED ISSUE #5982: Circle CI failed. Failed case is TestDecodeColumnValue.
FAIL: http_handler_test.go:335: HTTPHandlerTestSuite.TestDecodeColumnValue

See more https://circleci.com/gh/pingcap/tidb/16652?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

### PR DESCRIPTION
**Pull request by zimulala**
Make sure the rowBin with slashes can be handled.
e.g.
 rowBin=`CAIIyAEIBAIGYWJjCAYGAQCBCAgJ7/yY8LKF1M8Z`

Fix #5982

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @jackysp @zz-jason @AndreMouche

File: server/http_handler.go [hunk 1]
```diff
@@ -342,8 +342,25 @@ func (vh valueHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
...
-	bin := params[pRowBin]
+	// Get the unchanged binary.
+	if req.URL == nil {
+		err = errors.BadRequestf("Invalid URL")
+		writeError(w, err)
+		return
+	}
+	values := make(url.Values)
+	shouldUnescape := false
+	err = parseQuery(req.URL.RawQuery, values, shouldUnescape)
```
Comment (jackysp) [inline server/http_handler.go]: how about pass false directly?

File: docs/tidb_http_api.md [hunk 2]
```diff
@@ -96,9 +96,11 @@ timezone.*
...
     ```
 
-1. Get the column value by an encoded row and some information that can be obtained from a column of the table schema information.
+1. Get the column value by an encoded row and some information that can be obtained from a column of the table schema information. 
+
+    Argument example: rowBin=base64_encoded_row_value
 
     ```shell
-    curl http://{TiDBIP}:10080/tables/{colID}/{colFlag}/{colLen}/{rowBin}
+    curl http://{TiDBIP}:10080/tables/{colID}/{colFlag}/{colLen}?${rowBin}={val}
```
Comment (jackysp) [inline docs/tidb_http_api.md]: s/${rowBin}/rowBin/

File: server/http_handler.go (hunk 1 above)
Comment (zimulala) [reply server/http_handler.go]: I want it clearer.

**Comment by zimulala** [comment]
PTAL @jackysp

**Comment by shenli** [comment]
LGTM

**Pull request merged**